### ISSUE TITLE
Hi! I'm trying to fine-tuning yolo8vn.pt(based on coco-model) with my editional-images(+new tag)

### ISSUE THREAD

**Issue by OneChampOnly** [ultralytics/ultralytics#17472]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I would like to train yolo8vn.pt, which can detect 80 objects pre-trained with the existing coco dataset, using the images and examples I have (tag: "New_TAG"). However, the Catastrophic Forgetting phenomenon keeps occurring here, making it impossible to detect existing objects and extract tags, as well as errors such as detecting existing objects by classifying them as "New_TAG".
Could you please suggest a solution to this problem?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @OneChampOnly, thank you for reaching out and sharing your interest in using Ultralytics 🚀! 

To address issues like Catastrophic Forgetting during fine-tuning of your YOLO model, we recommend consulting the [Docs](https://docs.ultralytics.com). This includes sections on [Model Training Tips](https://docs.ultralytics.com/guides/model-training-tips/) that might help improve your training strategy.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) which is crucial for us to diagnose the issue accurately.

For general inquiries or if you have specific questions about custom training ❓, please include extensive details such as dataset examples and any training logs you can share. Ensuring you're following our training tips can also reduce such phenomena.

Join our vibrant Ultralytics community for more interactive support and insights! For real-time assistance, check out our [Discord](https://ultralytics.com/discord) 🎧, or if you prefer comprehensive discussions, our [Discourse](https://community.ultralytics.com) is the place to be. Dive into ongoing discussions on our [Subreddit](https://reddit.com/r/ultralytics) to exchange knowledge with fellow users.

## Upgrade

Make sure you're using the latest `ultralytics` package along with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml). The software is continuously improved, so having the latest version is essential:

```bash
pip install -U ultralytics
```

## Environments

For running YOLO, consider these verified environments with the necessary dependencies like [CUDA](https://developer.nvidia.com/cuda), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/):

- **Notebooks**: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud**: Check our [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon**: Visit our [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker**: Learn more in our [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

The green status badge denotes that all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing. These tests confirm the correct functionality of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu regularly.

This is an automated response, but rest assured that an Ultralytics engineer will review your issue and offer further assistance soon. Thank you for your patience! 🛠️

**Comment by Y-T-G** [comment]
It will always forget it if you don't provide images for existing classes.

You can try this
https://y-t-g.github.io/tutorials/yolov8n-add-classes/
